scripts/pi-hole/js/utils.js

```diff
@@ -378,10 +378,20 @@ function addTD(content) {
   return "<td>" + content + "</td> ";
 }
 
+function toPercent(number, fractionDigits = 0) {
+  const userLocale = navigator.language || "en-US";
+  return new Intl.NumberFormat(userLocale, {
+    style: "percent",
+    minimumFractionDigits: fractionDigits,
+    maximumFractionDigits: fractionDigits,
+  }).format(number / 100);
+}
+
 function colorBar(percentage, total, cssClass) {
-  var title = percentage.toFixed(1) + "% of " + total;
-  var bar = '<div class="progress-bar ' + cssClass + '" style="width: ' + percentage + '%"></div>';
-  return '<div class="progress progress-sm" title="' + title + '"> ' + bar + " </div>";
+  const percentFormatted = toPercent(percentage, 1);
```

Comment:
```suggestion
  const formattedPercentage= toPercent(percentage, 1);
```

Just a really niggly suggestion from me, can we use `formattedPercentage` (and on the next line) so that it is consitent with the usage in `index.js`?

Reply:
Sure thing! Fixed.